lib/dialogs/add-comment.js

```diff
@@ -0,0 +1,38 @@
+
+module.exports = (message, channel, team) => {
+  const linkToMessage =
+    `https://${team.domain}.slack.com/archives/${channel.id}/p${message.ts.replace('.', '')}`;
+
+  const comment = `> ${message.text}\n<sub>[View message in Slack](${linkToMessage})</sub>`;
```

Comment:
> View message in Slack

What do you think about just `View in Slack`?

I also wouldn't be opposed to getting rid of the `<sub>` to make the dialog that appears in Slack a little cleaner, but don't feel strongly about it.

Reply:
A nice future improvement would be to include the original author name. Something like `@wilhelmklopp said:`. This would require mapping Slack <=> GitHub usernames.

Reply:
> What do you think about just View in Slack?

I like that!

> I also wouldn't be opposed to getting rid of the <sub> to make the dialog that appears in Slack a little cleaner, but don't feel strongly about it.

Yeah, I keep coming back to re-consider the decision to have this visible to the user. There's some implementation difficulty with passing the value of the message URL along, though, so electing to keep it visible to the user for now.

> A nice future improvement would be to include the original author name. Something like @wilhelmklopp said:. This would require mapping Slack <=> GitHub usernames.

Hell yeah! We could do this today for folks who've connected their GitHub account, it just requires an additional API call. I'll open an issue for this once we've merged this PR